pyproject.toml

```diff
@@ -92,32 +95,108 @@ docs = [
     "nbsphinx>=0.7,<0.8.7"
 ]
 
+[tool.hatch.build.targets.wheel]
+packages = ["scvelo"]
+
+[tool.coverage.run]
+source = ["scvelo"]
+omit = [
+    "**/test_*.py",
+]
+
+[tool.pytest.ini_options]
+testpaths = ["tests"]
+xfail_strict = true
+
 [tool.black]
 include = '\.pyi?$'
 exclude = '''
-/(
-    \.git
-  | \.hg
-  | \.mypy_cache
-  | \.tox
-  | \.venv
-  | _build
-  | buck-out
-  | build
-  | dist
-)/
+(
+  /(
+      \.eggs
+    | \.git
+    | \.hg
+    | \.mypy_cache
+    | \.tox
+    | \.venv
+    | _build
+    | buck-out
+    | build
+    | dist
+  )/
+)
 '''
 
-[tool.isort]
-profile = "black"
-use_parentheses = true
-known_num = "networkx,numpy,pandas,scipy,sklearn,statmodels"
-known_plot = "matplotlib,mpl_toolkits,seaborn"
-known_bio = "anndata,scanpy"
-sections = "FUTURE,STDLIB,THIRDPARTY,NUM,PLOT,BIO,FIRSTPARTY,LOCALFOLDER"
-no_lines_before = "LOCALFOLDER"
-balanced_wrapping = true
-length_sort = "0"
-indent = "    "
-float_to_top = true
-order_by_type = false
```

Comment:
Is there a reason why you did not include the `isort` settings? I presume it's simply because it is not included in the cookiecutter TOML file? I personally prefer these settings since they make the import much more readable, IMO. What's the scverse consensus? Should everyone use the same settings or can we deviate from them a bit?